src/main/scala/chisel3/interface/Interface.scala

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: Apache-2.0
+package chisel3.interface
```

Comment:
I would prefer putting this in `chisel3.experimental.interface` at least while we iterate on the API

Reply:
Discussed with you offline. I'd like to kill `experimental` for new features going forward. We should also promote everything out of there that's currently in there.